eq-author/src/App/page/Design/Validation/AnswerValidation.js

```diff
@@ -179,180 +166,88 @@ class AnswerValidation extends React.PureComponent {
+      </ValidationButton>
+    );
 
-      validationButtons.push(rangeButtons);
-      validationButtons.push(rangeError);
+    if (type.id === "earliestDate" || type.id === "minDuration") {
+      return; // Don't display anything after the earliest date / min duration buttons - show after section
+    }
 
-      validationButtons.push(durationButtons);
-      validationButtons.push(durationError);
+    if (type.id === "maxDuration") {
+      validationMessage = DURATION_ERROR_MESSAGE;
+    }
+    else if (type.id === "latestDate") {
+      validationMessage = EARLIEST_BEFORE_LATEST_DATE;
+    }
+    else if (answer.type === "Date") {
+      validationMessage = EARLIEST_BEFORE_LATEST_DATE;
+    }
+    else if (find(validationErrors, error => error.errorCode.include("ERR_NO_VALUE"))) {
+      validationMessage = ERR_NO_VALUE;
     } else {
-      let validationMessage = null;
-
-      validationButtons = this.renderValidation(
-        validValidationTypes,
-        answer,
-        validationErrors
-      );
-
-      if (answer.type === "Date") {
-        validationMessage = EARLIEST_BEFORE_LATEST_DATE
-      } else {
-        const noValError = find(validationErrors.other, error =>
-          error.errorCode.includes("ERR_NO_VALUE")
-        );
-
-        const maxError = find(validationErrors.other, error =>
-          error.errorCode.includes("ERR_MIN_LARGER_THAN_MAX")
-        );
-        if (noValError || (noValError && maxError)) {
-          validationMessage = ERR_NO_VALUE
-        } else {
-          validationMessage = MAX_GREATER_THAN_MIN
-        }
-      }
-
-      const propertyError = this.renderPropertyError(
-        validationErrors.other.length > 0,
-        validationMessage
-      );
-      validationButtons.push(propertyError);
+      validationMessage = MAX_GREATER_THAN_MIN;
     }
 
-    return (
-      <ValidationContext.Provider value={{ answer }}>
-        {validationButtons}
-        <ModalWithNav
-          id={this.modalId}
-          onClose={this.handleModalClose}
-          navItems={validValidationTypes}
-          title={`${startCase(answer.type)} validation`}
-          isOpen={this.state.modalIsOpen}
-          startingTabId={this.state.startingTabId}
-        />
-      </ValidationContext.Provider>
-    );
-  }
+    validationButtons.push(renderPropertyError(
+      errors.length > 0,
+      validationMessage
+    ));
+  });
+
+  return (
+    <ValidationContext.Provider value={{ answer }}>
+      {validationButtons}
+      <ModalWithNav
+        id={modalId}
+        onClose={handleModalClose}
+        navItems={validValidationTypes}
+        title={`${startCase(answer.type)} validation`}
+        isOpen={modalIsOpen}
+        startingTabId={startingTabId}
+      />
+    </ValidationContext.Provider>
+  );
 }
 
 AnswerValidation.propTypes = {
   answer: CustomPropTypes.answer,
 };
 
-export default AnswerValidation;
+// Use memoisation to replace previous use of PureComponent
```

Comment:
```suggestion
```

When it comes to showing why you've changed something, try to keep that within the description of a PR instead of in comments within the code. In this example, a few months down the line the comment probably wont mean much as we'll likely forget we used PureComponent here. Nice one at pointing out the change of style, all the same. It's good to make sure people are aware of these things.